SPECS/redis/redis.spec

```diff
@@ -1,22 +1,22 @@
-Summary:	advanced key-value store
-Name:		redis
-Version:	5.0.5
-Release:        3%{?dist}
-License:	BSD
-URL:		http://redis.io/
-Group:		Applications/Databases
+Summary:	    advanced key-value store
+Name:		    redis
+Version:	    5.0.5
+Release:        4%{?dist}
+License:	    BSD
+URL:		    http://redis.io/
```

Comment:
Change the URL and Source0 "http" to "https"

Reply:
Fixed